frame/staking/src/lib.rs

```diff
@@ -819,8 +896,49 @@ decl_module! {
 
 		fn deposit_event() = default;
 
-		fn on_initialize() {
-			Self::ensure_storage_upgraded();
+		fn on_initialize(now: T::BlockNumber) {
+			if
```

Comment:
`Self::ensure_storage_upgraded()` call got lost, is that intentional?

Reply:
Darn. that was a merge miss. Will return.